### PR TITLE
Add Ironic multi-tenancy support. Fix bsc#1098145

### PR DESCRIPTION
**Pull request by dmpop**
Added info on Ironic multi-tenancy support.

### CODE REVIEW

File: xml/planning-architecture-example_configurations.xml [hunk 1]
```diff
@@ -68,6 +68,24 @@
...
+   networks. The interaction between &o_netw; and the physical switch is
+   facilitated by &o_netw;'s Modular Layer 2 (ML2) plugin. The &o_netw; ML2
+   plugin supports drivers to interact with various networks, as each vendor
+   may have their own extensions. Those drivers are referred to as &o_netw; ML2
+   mechanism drivers, or simply mechanism drivers.
+  </para>
+  <para>
+   The &o_iron; multi-tenancy feature has been validated using &ostack;
+   genericswitch mechanism driver. However, if the given physical switch
+   require a different mechanism driver, you must update the input model
```
Comment (sknorr) [inline xml/planning-architecture-example_configurations.xml]: require_s_

File: xml/planning-architecture-example_configurations.xml [hunk 2]
```diff
@@ -68,6 +68,24 @@
...
+   accordingly. To update the input model with a custom ML2 mechanism driver,
+   add specify the relevant information in the
```
Comment (sknorr) [inline xml/planning-architecture-example_configurations.xml]: either *add* or *specify*. decide!
also, any information on what exactly is "the relevant information"? e.g. some kind of example?

File: xml/planning-architecture-example_configurations.xml [hunk 3]
```diff
@@ -68,6 +68,24 @@
...
    </listitem>
   </itemizedlist>
+
+  <para>
+   &o_iron; multi-tenancy feature uses &o_netw; to manage the tenant
+   networks. The interaction between &o_netw; and the physical switch is
+   facilitated by &o_netw;'s Modular Layer 2 (ML2) plugin. The &o_netw; ML2
+   plugin supports drivers to interact with various networks, as each vendor
+   may have their own extensions. Those drivers are referred to as &o_netw; ML2
+   mechanism drivers, or simply mechanism drivers.
```
Comment (sknorr) [inline xml/planning-architecture-example_configurations.xml]: maybe add emphases around *ML2 mechanism drivers* and *mechanism drivers*.

File: xml/planning-architecture-example_configurations.xml [hunk 4]
```diff
@@ -68,6 +68,24 @@
     </para>
    </listitem>
   </itemizedlist>
+
+  <para>
+   &o_iron; multi-tenancy feature uses &o_netw; to manage the tenant
```
Comment (sknorr) [inline xml/planning-architecture-example_configurations.xml]: "*The* Ironic multi-tenancy feature ..." ..? [NB: i have not checked the definition of \&o_iron;, if the *The* makes no sense, just ignore me on this one.]

File: xml/planning-architecture-example_configurations.xml (hunk 2 above)
Comment (dmpop) [reply xml/planning-architecture-example_configurations.xml]: Can we let the readers to decide which one they like best: add or specify?

The bug reporter probably assumes that if you know what multi-tenancy is, it's highly likely that you also know what "relevant info" is. So I'm inclined to leave it as it is.

File: xml/planning-architecture-example_configurations.xml (hunk 3 above)
Comment (dmpop) [reply xml/planning-architecture-example_configurations.xml]: Wird gemacht!

File: xml/planning-architecture-example_configurations.xml (hunk 4 above)
Comment (dmpop) [reply xml/planning-architecture-example_configurations.xml]: I choose to ignore you on this one. My warm feelings towards you remain unchanged, though.

**Pull request merged**